Mathlib/Topology/Algebra/Order/MonotoneConvergence.lean

```diff
@@ -310,6 +310,12 @@ theorem iInf_eq_of_tendsto {α} [TopologicalSpace α] [CompleteLinearOrder α] [
   tendsto_nhds_unique (tendsto_atTop_iInf hf)
 #align infi_eq_of_tendsto iInf_eq_of_tendsto
 
+theorem tendsto_of_antitone {ι α : Type*} [Preorder ι] [TopologicalSpace α]
```

Comment:
Can you put that next to `tendsto_of_monotone`? Or is there a good reason for putting it here instead?

Reply:
That makes, sense, done!